cli/config_file.rs

```diff
@@ -227,6 +228,56 @@ pub fn json_merge(a: &mut Value, b: &Value) {
   }
 }
 
+/// Based on flags and an optional configuration file, return a resolved module
+/// specifier to an import map.
+pub fn resolve_import_map_specifier(
+  flags: &flags::Flags,
```

Comment:
Could we take `flag.import_map_path` as `maybe_import_map_path` instead? That way we wouldn't have to import `flags` module here - seems preferable not to couple the two